frontend/src/pages/ErrorsV2/ErrorInstance/ErrorInstance.tsx

```diff
@@ -329,15 +331,15 @@ const ErrorInstance: React.FC<Props> = ({ errorGroup }) => {
 								}
 								if (errorObject.session?.secure_id) {
 									queryParams.push({
-										key: 'secure_session_id',
+										key: ReservedLogKey.SecureSessionId,
```

Comment:
This is referencing these enums: 
https://github.com/highlight/highlight/blob/756e9ad1ee1446bd04437ba7eba18fe9a1af165f/backend/private-graph/graph/schema.graphqls#L538-L546